src/BenchmarksDriver/Serializers/SignalRSerializer.cs

```diff
@@ -195,6 +198,7 @@ class SignalRSerializer : IResultsSerializer
                     p.AddWithValue("@WebHost", serverJob.WebHost.ToString());
                     p.AddWithValue("@Transport", clientJob.ClientProperties["TransportType"]);
                     p.AddWithValue("@HubProtocol", clientJob.ClientProperties["HubProtocol"]);
+                    p.AddWithValue("@ClientProperties", string.Join(',', clientJob.ClientProperties.ToArray()));
```

Comment:
That, or a json serialized array in case you need to process it programatically ?

Reply:
Or a json document, with property names.

Reply:
`string.Split` not good enough?

Reply:
There is no escaping, so no guaranty you can deserialize it with a split.